### DISCUSSION TITLE
blog/i-was-using-ai-tools-wrong-and-maybe-youre-too/

### DISCUSSION THREAD

**Discussion by giscus** [Jitesh117/blog_comments#20] [Q&A]
# blog/i-was-using-ai-tools-wrong-and-maybe-youre-too/

“This quote is here to trick you into thinking this article is profound.”
- Me
AI tools like ChatGPT and Claude are basically the new shiny toys every developer has. When they first rolled out, I wasn’t too interested. But since July of last year, things escalated quickly. I went from “meh, I’ll Google it” to “GPT, save me!” in record time.
The Excitement of AI Assistance When I first gave GPT a spin, it felt like magic.

https://jitesh117.github.io/blog/i-was-using-ai-tools-wrong-and-maybe-youre-too/

**Comment by HodBadichi** [comment]
Great write up
